scripts/deploy.js

```diff
@@ -9,14 +9,23 @@ const path = require('path');
 const fs = require("fs");
 const fse = require("fs-extra");
 
+const VERSION = require(path.resolve(__dirname, '../package.json')).version;
+
 const remoteUrl = "https://github.com/molstar/molstar.github.io.git";
+const dataDir = path.resolve(__dirname, '../data/');
 const buildDir = path.resolve(__dirname, '../build/');
 const deployDir = path.resolve(buildDir, 'deploy/');
 const localPath = path.resolve(deployDir, 'molstar.github.io/');
 
 const analyticsTag = /<!-- __MOLSTAR_ANALYTICS__ -->/g;
 const analyticsCode = `<!-- Cloudflare Web Analytics --><script defer src='https://static.cloudflareinsights.com/beacon.min.js' data-cf-beacon='{"token": "c414cbae2d284ea995171a81e4a3e721"}'></script><!-- End Cloudflare Web Analytics --><iframe src="https://web3dsurvey.com/collector-iframe.html" style="width: 1px; height: 1px;"></iframe>`;
 
+const manifestTag = /<!-- __MOLSTAR_MANIFEST__ -->/g;
```

Comment:
I've only now realized that this deploy script added the tracking to the viewer. I've usually been doing this manually, meaning for some versions of the viewer the analytics wasn't present.

I think we should add a version of this script that just modifies the files, but doesn't do the automatic push to GH Pages. I my case, I sometimes only want to update a particular page or demo for example. I can have a look into this in a separate PR.